Task1.java

```diff
@@ -0,0 +1,24 @@
+public class Task1 {
+

```

Comment:
You did a good job on refactoring tasks 1-4! Well done!
However, nothing has changed for task 5. I would ask you to rewrite and simplify it, as I'm sure that it's possible. It might be a matter of time, but the more you practice, the more confident and experienced you get.